.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# The following users are the maintainers of all openedx-events samples repositories
+* @mariajgrimaldi
```

Comment:
@felipemontoya: Here, I added myself as one of the framework's maintainers, although I'm not sure if tagging a user directly would be the best approach. Should we create an internal group for hooks maintainership, or should it be handled by another team?

Reply:
This repo is small enough that I don't mind tagging users directly. If you definitely prefer having a group, let me know and I'll create it. For the moment I think this is fine and can be merged.